assets/app/views/browse/build-config.html

```diff
@@ -55,7 +55,14 @@
                   <dd>{{buildConfig.spec.output.to | imageObjectRef : buildConfig.metadata.namespace}}</dd>
                 </div>
               </dl>
-            </div>
+              <div ng-if="(buildConfig | buildStrategy).env">
+                <h3>Environment Variables</h3>
+                <dl class="dl-horizontal left">
+                  <dt ng-repeat-start="envVar in (buildConfig | buildStrategy).env">{{envVar.name}}:</dt>
+                  <dd ng-repeat-end>{{envVar.value}}</dd>
+                </dl>
+              </div>               
```

Comment:
Hmmm the build details page never got split into sections, let me do that
now, otherwise the env var section will look out of place.

On Fri, Nov 13, 2015 at 2:09 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/browse/build-config.html
> <https://github.com/openshift/origin/pull/5893#discussion_r44820971>:
>
> > @@ -55,7 +55,14 @@
> >                    <dd>{{buildConfig.spec.output.to | imageObjectRef : buildConfig.metadata.namespace}}</dd>
> >                  </div>
> >                </dl>
> > -            </div>
> > +              <div ng-if="(buildConfig | buildStrategy).env">
> > +                <h3>Environment Variables</h3>
> > +                <dl class="dl-horizontal left">
> > +                  <dt ng-repeat-start="envVar in (buildConfig | buildStrategy).env">{{envVar.name}}:</dt>
> > +                  <dd ng-repeat-end>{{envVar.value}}</dd>
> > +                </dl>
> > +              </div>
>
> Should this be in the build page, too?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5893/files#r44820971>.
>